src/panels/lovelace/elements/hui-state-badge-element.ts

```diff
@@ -1,21 +1,29 @@
-import { html, LitElement, TemplateResult } from "lit-element";
+import {
+  html,
+  LitElement,
+  TemplateResult,
+  customElement,
+  property,
+} from "lit-element";
 
 import "../../../components/entity/ha-state-label-badge";
+import "../components/hui-warning";
 
 import computeStateName from "../../../common/entity/compute_state_name";
 import { LovelaceElement, LovelaceElementConfig } from "./types";
 import { HomeAssistant } from "../../../types";
 
+export interface Config extends LovelaceElementConfig {
```

Comment:
I think that we should pick unique names for exported constants, so have `StateBadgeElementConfig`

Reply:
Go ahead and merge this one and I'll swing back around and do that for ALL configs